src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,21 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier(new ColorSupplier());
 
+        Figure[] figures = new Figure[4];
```

Comment:
Looks like a [magic number](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants).